packages/common/config/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../config/tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "../lib"
+  },
+  "include": ["../src/**/*"],
+  "exclude": ["../src/**/__tests__/**/*", "../src/**/__mocks__/**/*"]
+}
```

Comment:
Shouldn't `packages/common` have a `tsconfig.json` file at its root, rather than nested in `packages/common/config`? Seems awkward to have so many `../`s here, and the `tsconfig.json` file is what defines the root directory of a TypeScript project (or so I thought).

Reply:
@benjamn I like the idea of keeping all project config related files in a common location, outside of the project root, hence the `/config` directory (which houses the Jest, Rollup, TS, etc. configs). Yes, `tsconfig` defines the root, which is why the `include`s / `exclude`'s are overridden. I'm not overly attached to this approach though, so if you'd prefer `tsconfig` in the root, then no problemo!